extension.driver.php

```diff
@@ -107,6 +107,14 @@ public function update($previousVersion = false)
                 );
             }
         }
+
+        if (version_compare($previousVersion, '2.0.0', '<=')) {
+            Symphony::Database()->query(
+                "ALTER TABLE `tbl_fields_hashid_field`
+                    DROP COLUMN `size`;"
+             );
+        }
+
```

Comment:
Please create the `2.0.1` version in the meta.xml file and change this line to

```php
if (version_compare($previousVersion, '2.0.1', '<')) {
```

Go back and manually add the column in your test database in order to test the upgrade script again. Thanks!